DesignspaceEditor2.roboFontExt/lib/designspaceEditor/ui.py

```diff
@@ -558,16 +558,61 @@ class InstancesAttributesPopover(BaseAttributePopover):
     def build(self, item):
         """
         support:
+            * postScriptFontName
+            * styleMapFamilyName
+            * styleMapStyleName
             * localisedFamilyName
             * localisedStyleName
         """
+        # Create the instancesDescriptor attribute
         self.instancesDescriptor = item["object"]
 
-        self.popover.tabs = vanilla.Tabs((0, 15, -0, -0), ["Localised Family Name", "Localised Style Name"])
+        # Create the popover tabs
+        self.popover.tabs = vanilla.Tabs((0, 15, -0, -0), ["Additional Names", "Localised Family Name", "Localised Style Name"])
 
-        self.instanceLocalisedFamilyName = self.popover.tabs[0]
-        self.instanceLocalisedStyleName = self.popover.tabs[1]
+        # Assign the tabs to variables for easier access
+        self.instanceStyleMapNames = self.popover.tabs[0]
+        self.instanceLocalisedFamilyName = self.popover.tabs[1]
+        self.instanceLocalisedStyleName = self.popover.tabs[2]
 
+        # Create the container for the Style Map tab
+        self.instanceStyleMapNames.container = vanilla.Box((10, 10, -10, -10))
+        self.styleMapStyleOptions = ["regular", "italic", "bold", "bold italic"]
+
+        col1 = 10
+        col2 = 180
+        padding = 24
+        y = 10
+
+        # Create the text box and edit box for identifier Name
+        self.instanceStyleMapNames.container.identifierNameLabel = vanilla.TextBox((col1, y, col2-padding, 22), "Identifying Name:", alignment="right")
+        self.instanceStyleMapNames.container.identifierNameTextBox = vanilla.EditText((col2, y, -40, 22), "", callback=self.controlEditCallback)
+        self.instanceStyleMapNames.container.identifierNameAutoBtn = vanilla.SquareButton((-32, y, -10, 22), "􀜍", callback=self.autoIdentifyingNameCallback)
```

Comment:
that unicode value will only work on systems where SF Symbols is installed and preferable this has to be an image from an SF symbol, see https://github.com/LettError/designSpaceRoboFontExtension/blob/e3c94c796cac46b1152c57bcf1f3b967ec0b0c52/DesignspaceEditor2.roboFontExt/lib/designspaceEditor/ui.py#L778 and add the SF Symbol as an asset in the [resource](https://github.com/LettError/designSpaceRoboFontExtension/blob/e3c94c796cac46b1152c57bcf1f3b967ec0b0c52/DesignspaceEditor2.roboFontExt/resources/toolbar_30_30_smallcircle.filled.circle.fill.pdf) folder

Reply:
Done.